docs/01-product.md

```
@@ -0,0 +1,312 @@
+Long-term or complex injuries can generate large amounts of fragmented information:
+
+- Symptoms
+- Treatments and outcomes
+- Medical visits
+- Timeline events
+- Injury history
+
+As this information grows, it becomes difficult to quickly answer questions such as:
+
+> What treatments have I tried?
+
```

markdownlint-cli2 (0.23.2)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)